packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -80,21 +81,19 @@ _ref:
             The config object has the following properties:
 
             - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
-            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
+            - `homePageId: string`: The id of the page that will load when a visitor navigates to the home route, for example, `yourdomain.com`.
 
             ## Auth
 
             [TODO]: <> (# TODO: Add auth things here)
 
             ## Global
 
-            Any data that you wish to use in your app can be stored in the __global__ object, and accessed using the [`_global`](/_global) operator. This is a good place to store data or configuration that is used throughout the app, for example the url of a logo or configuration of a page, since then these are only written once, and can be updated easily.
-
-            The global object can also be modified using the [`SetGlobal`](/SetGlobal) action.
+            In a Lowdefy app, you can define global variables in the __global__ object, which can be accessed using the [`_global`](/_global) operator, and modified using the [`SetGlobal`](/SetGlobal) action. This is a good place to store data or configuration that is used over various pages throughout the app, for example, user preferences that can be modified using `SetGlobal` on one page and easily referenced on all others.
 
             ## Connections
 
-            In a Lowdefy app you can integrate with other services like API's or databases using `connections` and `requests`. Connections configure the connection settings to the service, and often contain parameters like connection strings, urls and secrets such as passwords or API keys. Requests are used to interact with the connection, for example inserting a data record, executing a query or calling a API end-point.
+            In a Lowdefy app you can integrate with other services like API's or databases using `connections` and `requests`. Use connections to configure the settings to a service, for example, settings such as urls, authentication config or [`secrets`](/secrets) like passwords or API keys. Requests use connections to send and receive data from external services, for example, inserting a data record or executing a query on a database, or simply calling an API end-point.
```

Comment:
We can maybe do something a little better than 
"send and receive data from external services,"
To me this is more specific actions with the external service API, so i like the word "interactions" there. The request id the part of the connection that does the work. With custom plugins this can become quite complicated as well, you could write a whole piece of code that executes business logic there.

I guess it all boils down to sending and receiving data, but I don't think that captures the full idea

Reply:
Some requests like sending an email or processing a payment with Stripe might not feel like "sending data" to general users.

Reply:
Not sure if changing "send and receive data from external services" to "interact with external services" is better here. That's the current change that I have made in the latest commit. Let me know if you have better ideas.

Change made in 386463869c78c494c05636e1e435419c96917b02